README.md

```diff
@@ -295,7 +295,7 @@ Tested on a Raspberry Pi Zero with a clean install of minimal Raspbian Stretch (
     ```bash
     git clone https://github.com/monero-project/monero.git
     cd monero
-    git checkout tags/v0.17.2.0
+    git checkout tags/v0.17.2.3
```

Comment:
In line [225](https://github.com/monero-project/monero/pull/7805/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R225) for building under Ubuntu we have:

`git checkout release-v0.17`

I think it would be better to use the same line here for more consistency. It will also reduce some maintenance burden (changes will be required rarer)

Reply:
It doesn't matter really as I use search / replace anyway. It's inside a branch so it isn't visible by users on the repo README by default anyway.

Reply:
OK. Especially the 2nd sentence makes it minor.